### DISCUSSION TITLE
Add a new option that automatically adds a _created time stamp to each document when _rev is changed

### DISCUSSION THREAD

**Discussion by SinanGabel** [apache/couchdb#3098] [Ideas] [answered]
(1) What?

Every time a document has its _rev changed another key:value is updated, a _created time stamp with time in milliseconds since 1970.
```

{
_id: ...
_rev: ...
_created: 1598352308309
}
```

(2) Configuration option (choice) in local.ini

The configuration could e.g. be in the stats configuration part:
[configuration stats](https://docs.couchdb.org/en/stable/config/misc.html#statistic-calculation)

```
[stats]
doc_created = true
...
```

**Comment by wohali** [comment, marked as answer]
Duplicate of https://github.com/apache/couchdb/issues/1533
